examples/databinding/MODULE.bazel

```diff
@@ -0,0 +1,51 @@
+module(
+    name = "basicapp",
+)
+
+bazel_dep(name = "rules_java", version = "7.12.2")
+bazel_dep(name = "bazel_skylib", version = "1.3.0")
+bazel_dep(name = "rules_jvm_external", version = "6.6")
+
+bazel_dep(
+    name = "rules_android",
+    version = "0.6.0",
```

Comment:
Let's change this to 0.6.3 (doesn't really matter due to local_path_override, but will be consistent with HEAD of main branch)